Отправляю 7 лабу

PihterovaNatalia opened the pull request.

PihterovaNatalia committed:
Отправляю 7 лабу

PihterovaNatalia committed:
Add files via upload

PihterovaNatalia committed:
лаб 8

NemirovichMark commented:
Здравствуйте, Наталья.

В 7й лабе 1й и 2й номера принял. 3й нет. Складывает как-то неправильно.
![image](https://github.com/NemirovichMark/BBI-23-3/assets/112826355/168cfc95-4c92-492c-ab1f-bebbc2b44c15)

В 8й лабораторной 7й номер не работает. Остальные принял.

На доработку.

PihterovaNatalia committed:
Add files via upload

PihterovaNatalia committed:
8 лабораторная доработанная 

8 лабораторная доработанная

PihterovaNatalia committed:
Add files via upload

8 лабораторная доработанная

NemirovichMark commented:
Здравствуйте, Наталья.
КР
15 / 20
1й номер
-1 публичные поля - нарушение ООП
-1 Unhandled exception. System.DivideByZeroException: Деление на ноль невозможно.
2й номер
-1 не запускается
-1 умножение не работает
-1 деление не работает

8ая лабораторная
Ошибка в строке 178, поэтому вывод пустой
if (words.Contains(sub))

На доработку.

PihterovaNatalia committed:
7 лабораторная 3 номер

Здравствуйте!) Вы отправляли мне 3 номер на доработку, но я не совсем понимаю что не так , по сути  должен считать правильно. Так как в массиве quest  записаны ответы такие же как для  России и Японии.

PihterovaNatalia committed:
9 лабораторная

NemirovichMark commented:
Здравствуйте, Наталья.
7ю принял
8ая лабораторная:
11й номер - написать свою сортировку и компаратор (сравнение строк)
    foreach (var surname in words.OrderBy(x => x))
    {
        res += $"{surname}, ";
    }
9ая. Ошибка при чтении:
![image](https://github.com/NemirovichMark/BBI-23-3/assets/112826355/e798c076-78b6-40d6-b1b4-ccf470d6bd0f)
![image](https://github.com/NemirovichMark/BBI-23-3/assets/112826355/625610d4-6728-4f0e-9f06-f5513edef488)
![image](https://github.com/NemirovichMark/BBI-23-3/assets/112826355/6520acd7-4ff0-4fb8-8e4b-2d8a157ae98c)

На доработку.

PihterovaNatalia committed:
9 лабораторная исправленная

NemirovichMark commented:
Здравствуйте, Наталья.
9ю принял.
